openapi-src/V2/paths/Forms/post-v2-forms-entity-form-uuid.yml

```diff
@@ -13,7 +13,7 @@ parameters:
     name: ENTITY
     in: path
     required: true
-    description: allowed values projects/sites/nurseries/project-reports/site-reports/nursery-reports
+    description: allowed values projects/sites/nurseries
```

Comment:
Creating reports via form isn't supported, and wasn't being used by the FE. This is simply a documentation update to reflect that fact.